starter-code/routes/movie.routes.js

```diff
@@ -0,0 +1,60 @@
+const express = require('express');
+const router  = express.Router();
+
+const MovieModel = require('../models/movie-model')
+
+/* GET home page */
+router.get('/movies', (req, res, next) => {
+  console.log("Hey its the movie page")
+    MovieModel.find()
+      .then((movies) => {
+        console.log(movies)
+
+        res.render('movies/index.hbs', {movies})
+      })
+      .catch((err) => {
+        console.log('Oh-no!', err)
+        next()
+      })
+  });
+
+  router.get('/movies/:id', (req, res, next) => {
+    let myId = req.params.id
+    
+    MovieModel.findById(myId)
+      .then((result) => {
+        console.log(result)
```

Comment:
instead of generic variables like result, data and response, try to name your variables with more descriptive names like oneMovie or movieObj